### PR TITLE
[CoreBundle] Add option to link assets from current theme through controller

### PR DESCRIPTION
**Pull request by ahilles107**
(without knowing current organization or tenant codes)

| Q                         | A
| ------------------------- | ---
| Bug fix?                  | no
| New feature?              | yes
| BC breaks?                | no
| Deprecations?             | no
| Tests pass?               | yes
| Changelog update required | yes
| Fixed tickets             | [SWP-327](https://dev.sourcefabric.org/browse/SWP-327)
| License                   | AGPLv3

### CODE REVIEW

File: docs/themes.rst [hunk 1]
```diff
@@ -108,6 +108,33 @@ Load assets from the public ``web`` directory
...
+Where {filePath} is path for your file from public directory inside theme.
+
+Load Service Worker files (from domain root level)
+``````````````````````````````````````````````````
+
+If You want to use service worker or manifest file (it must be placed in root level) then you can use this url:
+
+.. code-block:: twig
+
+    /public/{fileName}.{fileExtension}
```
Comment (takeit) [inline docs/themes.rst]: shouldn't this example be `/public/{fileName}.{fileExtension}` according to its description?

File: src/SWP/Bundle/CoreBundle/Controller/StaticThemeAssetsController.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+namespace SWP\Bundle\CoreBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\ResponseHeaderBag;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class StaticThemeAssetsController extends Controller
```
Comment (takeit) [inline src/SWP/Bundle/CoreBundle/Controller/StaticThemeAssetsController.php]: tests are missing

File: docs/themes.rst [hunk 3]
```diff
@@ -108,6 +108,33 @@ Load assets from the public ``web`` directory
...
+    /public/{filePath}
+
+    ex. <link rel="stylesheet" href="/public/css/example.css" />
+
+Where {filePath} is path for your file from public directory inside theme.
+
+Load Service Worker files (from domain root level)
+``````````````````````````````````````````````````
+
+If You want to use service worker or manifest file (it must be placed in root level) then you can use this url:
```
Comment (takeit) [inline docs/themes.rst]: I would link to service worker definition here

File: src/SWP/Bundle/CoreBundle/Controller/StaticThemeAssetsController.php (hunk 2 above)
Comment (ahilles107) [inline src/SWP/Bundle/CoreBundle/Controller/StaticThemeAssetsController.php]: tests are added

**Pull request merged**